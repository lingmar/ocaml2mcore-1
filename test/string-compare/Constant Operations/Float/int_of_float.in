int_of_float 9.990000
