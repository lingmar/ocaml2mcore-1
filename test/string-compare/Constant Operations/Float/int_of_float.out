mexpr
floorfi 9.990000
